tests/test_templates.py

```diff
@@ -37,7 +37,6 @@ def test_vector(self):
   <head><title>Found Object(s)</title></head>
   <body>
     <div class="results">
-
```

Comment:
If we're adding a check to skip lines, can we just leave the original newlines in place?